code/game/objects/items/miscellaneous.dm

```diff
@@ -19,3 +19,66 @@
 	icon_state = "skub"
 	w_class = WEIGHT_CLASS_BULKY
 	attack_verb = list("skubbed")
+	
+/obj/item/herobeacon
+	name = "heroic beacon"
+	desc = "To summon heroes from the past to protect the future"
```

Comment:
punctuation reee

Reply:
_wot's dat?_ (ok in all seriousness i haven't spotted it, real easy to fix, just point where you'd like it)

Reply:
needs a period